.github/workflows/pr-build.yml

```diff
@@ -42,6 +42,60 @@ jobs:
         with: 
           args: build -x avroCompatibilityCheck dockerBuildImages
 
+      - name: Run Trivy vulnerability scanner for ingester
```

Comment:
I was in two thoughts.
1) Only do scan for `ingester` image + the those are deployed separately
2) Do scan of all the publish images?

I have taken option 2). 
@aaron-steinfeld @ravisingal  Would that add significant build time?

Reply:
I'd do all of them, it shouldn't be that slow. If it's prohibitive, we can optimize after.